tox.ini

```
@@ -60,10 +60,14 @@ deps =
     oldestdeps: minimum_dependencies
     devdeps: numpy>=0.0.dev0
     devdeps: scipy>=0.0.dev0
+    devdeps: scikit-image>=0.0.dev0
     devdeps: pyerfa>=0.0.dev0
     devdeps: astropy>=0.0.dev0
-    devdeps: tweakreg>=0.0.dev0
     devdeps: requests>=0.0.dev0
+    devdeps: tweakreg>=0.0.dev0
```

```suggestion
    devdeps: tweakwcs>=0.0.dev0
```

I am sorry!! I typo'd this and now it's merged